### DISCUSSION TITLE
Show badges as in dock

### DISCUSSION THREAD

**Discussion by dipsy** [rxhanson/Charmstone-Community#42] [Ideas]
Example
![CleanShot 2024-02-04 at 21 37 04@2x](https://github.com/rxhanson/Charmstone-Community/assets/1336665/5024e2ad-f741-4416-8ee1-068cb8463f9b)

**Comment by rxhanson** [comment]
Thanks for the feedback. Unfortunately, there is no macOS API for badges. There might be a way to do this, but it's not ideal. Last I looked it would require something like polling the accessibility API to get the badge & count. This walks down a path that's sort of fragile, and in the case of Charmstone, I really like that it doesn't use any of the more fragile things in macOS.
Maybe at some point Apple will release an API for it?
